src/Chapter2.hs

```diff
@@ -336,7 +336,13 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b xs
+  | b >= a = []
```

Comment:
I think here you wanted to use a different comparison operator. When you call `subList from to` and `to` is lower than `from`, then the resulting list is empty
```suggestion
  | b < a = []
```